### DISCUSSION TITLE
Assessing differential co-accessibility across different cell populations

### DISCUSSION THREAD

**Discussion by Jayesh1411** [stuart-lab/signac#1280] [Q&A] [answered]
Hi @timoast, Thank you very much such a wonderful package. 
I was interested in assessing the differential co-accessibility across different cell populations using the LinkPlot() function, however it can only provide the aggregated co-accessibility for the entire object. It would be really helpful if you could provide your insight on how I can assess differential co-accessibility in the different cell populations as per a metadata column in the seurat object. 

Thank you very much for your time and consideration.

**Comment by timoast** [comment, marked as answer]
Hi, I think you should probably ask the Cicero developers this question, since this relates to the Cicero method rather than Signac itself.

Some related comments are in this discussion: https://github.com/stuart-lab/signac/discussions/949
